components/common/comment_icon.tsx

```diff
@@ -29,7 +29,7 @@ export default class CommentIcon extends React.PureComponent<Props> {
 
     public render(): JSX.Element {
         let commentCountSpan: JSX.Element | null = null;
-        let iconStyle = 'post-menu__item post-menu__item--wide post-menu__item--comment';
```

Comment:
This was the only reference to that class in the whole codebase, so, I've removed it